spec/classes/foreman_proxy__plugin__pulp_spec.rb

```diff
@@ -63,52 +39,22 @@
         it 'should remove pulp3.yml' do
           is_expected.to contain_file("#{etc_dir}/foreman-proxy/settings.d/pulp3.yml")
             .with_ensure('absent')
-	end
+	      end
       end
 
       describe 'with overrides' do
         let :params do
           {
-            pulpnode_enabled: true,
             pulpcore_enabled: true,
             pulpcore_mirror: true,
             pulpcore_api_url: 'https://pulpcore.example.com',
-            pulpcore_content_url: 'https://pulpcore.example.com/pulp/content',
-            pulp_url: 'https://pulp.example.com',
-            pulp_dir: '/tmp/pulp',
-            pulp_content_dir: '/tmp/content',
-            puppet_content_dir: '/tmp/puppet',
-            mongodb_dir: '/tmp/mongodb',
+            pulpcore_content_url: 'https://pulpcore.example.com/pulp/content'
```

Comment:
It's better to use a trailing comma:
```suggestion
            pulpcore_content_url: 'https://pulpcore.example.com/pulp/content',
```